contrib/automation_tests/orbit_loaded_top_down_bottom_up.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright (c) 2021 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.bottom_up_tab import VerifyBottomUpContentForLoadedCapture
+from test_cases.capture_window import FilterTracks, SelectAllCallstacks, ZoomOut
+from test_cases.connection_window import LoadCapture
+from test_cases.top_down_tab import VerifyTopDownContentForLoadedCapture
+"""Inspect the top-down view in Orbit using pywinauto.
+
+Before this script is run Orbit needs to be started.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test, it needs
+to be run from 64 bit python.
+
+This automation script covers a basic workflow:
+ - load a known capture
+ - verify the content of the top-down view
+ - verify the content of the bottom-up view
+ - select all callstacks using the "(all threads)" track
+ - verify the content of the top-down view for the selection
+ - verify the content of the bottom-up view for the selection
+"""
+
+
+def main(argv):
+    test_cases = [
+        LoadCapture(capture_file_path="testdata\\OrbitTest_1-72.orbit"),
+        VerifyTopDownContentForLoadedCapture(selection_tab=False),
+        VerifyBottomUpContentForLoadedCapture(selection_tab=False),
+
+        ZoomOut(),
+        FilterTracks(filter_string='threads{)}'),
```

Comment:
Why the curly braces?

Reply:
Ah, I wasn't aware of that. Might be worth extending the comment in `FilterTracks::_execute` to indicate this, but that's not really the scope of this PR, so it's fine as is.

Reply:
I added a note in `FilterTracks._execute`'s comment.